src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration status="info">
+    <Appenders>
+        <Console name="LogToConsole" target="SYSTEM_OUT">
+            <PatternLayout pattern="%d{HH:mm:ss.SSS} %-5level %c:%L - %msg%n"/>
+        </Console>
+        <File name="LogToFile" fileName="logs/app.log">
+            <PatternLayout>
+                <Pattern>%d %p %c:%L %m%n</Pattern>
+            </PatternLayout>
+        </File>
+    </Appenders>
+    <Loggers>
+        <Logger name="mate.academy" level="debug" additivity="false">
```

Comment:
In my opinion we should use info level, not debug
```suggestion
        <Logger name="mate.academy" level="info" additivity="false">
```

Reply:
I think it's an optional value ;)
I experimented a bit with these values. And I left it as it was since we are not forced anywhere in this project to use one or another level of logging)